codecov_cli/helpers/request.py

```diff
@@ -95,6 +96,15 @@ def get_token_header_or_fail(token: str) -> dict:
     return {"Authorization": f"token {token}"}
 
 
+def get_auth_header(token):
+    tokenless = os.environ.get("TOKENLESS", None)
```

Comment:
are we making this change for the orb and step too? Why doesn't the CLI just do this itself?

Reply:
i don't think the CLI has access to the same context as the action but i can double check, but you are correct we should make a similar change the the orb and step as well